src/poliastro/core/perturbations.py

```diff
@@ -218,6 +218,29 @@ def third_body(t0, state, k, k_third, perturbation_body):
         gravitational constant, (km^3/s^2)
     perturbation_body: a callable object returning the position of the pertubation body that causes the perturbation
 
+    Example
+    -------
+    For clarifying the nature of **perturbation_body** argument in **third_body()** function, we can look into the following example:
+
+    >>> from poliastro.ephem import build_ephem_interpolant
+    >>> from poliastro.bodies import Moon
+    >>> from astropy.time import Time, TimeDelta
+    >>> epoch = Time(
+    ...    2454283.0, format="jd", scale="tdb"
+    ...     )  # setting the exact event date is important
+    >>> body_r = build_ephem_interpolant(
+    ...   Moon, 28 * u.day, (epoch.value * u.day, epoch.value * u.day + 60 * u.day), rtol=1e-2
+    ... )
+    >>> type(body_r)
```

Comment:
Can we add an example of how to use the interpolant?

Reply:
We can add it but it would be very similar or else exactly what we have in the notebook. 🤔

Reply:
Do you still intend to work on this? Otherwise I can try to write something myself

Reply:
Yes, I can work upon this... Just need some Guidance for the same or the main essence of the example :)